codegen/src/test/resources/software/amazon/awssdk/codegen/poet/client/c2j/rpcv2/customization.config

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Nit: do we need customization.config and endpoint-rule-set files?

Reply:
removed.